src/Xamarin.Android.Build.Tasks/Xamarin.Android.Legacy.targets

```diff
@@ -258,14 +258,18 @@ projects. .NET 5 projects will not import this file.
           Include="@(ReferenceCopyLocalPaths)"
           Condition="'%(ReferenceCopyLocalPaths.ResolvedFrom)' != 'ImplicitlyExpandDesignTimeFacades' And '%(ReferenceCopyLocalPaths.Extension)' == '.dll' And '%(ReferenceCopyLocalPaths.RelativeDir)' == '' And Exists('%(ReferenceCopyLocalPaths.Identity)') "
       />
+      <FilteredAssemblies
+          Include="@(ReferenceDependencyPaths)"
+          Condition="'%(ReferenceDependencyPaths.ResolvedFrom)' != 'ImplicitlyExpandDesignTimeFacades' And Exists('%(ReferenceDependencyPaths.Identity)') "
+      />
```

Comment:
I think this needs to use `@(_ReferenceDependencyPaths)`:

https://github.com/xamarin/xamarin-android/blob/eb6ea0b55c89ce7e0b107c72900be9d019fb4b5b/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets#L438-L442

This way we will include the `%(AndroidSkip...)` metadata, it matches the use of `@(_ReferencePath)` below.

Reply:
@brendanzagaeski I think with this change ^ we can probably merge this. Its looking good.

Reply:
Ooo, nice catch. I've added that change now.